judge/management/commands/create_problem.py

```diff
@@ -12,12 +13,28 @@ def add_arguments(self, parser):
         parser.add_argument('body', help='problem description')
         parser.add_argument('type', help='problem type')
         parser.add_argument('group', help='problem group')
+        parser.add_argument('time_limit', help='time limit in (fractional) seconds')
+        parser.add_argument('memory_limit', help='memory limit in kilobytes')
+        parser.add_argument('points', help='problem points')
+        parser.add_argument('--partial-points', help='allow partial points',
+            action='store_true', default=False, dest='pr_pts')
+        parser.add_argument('--all-languages', help='allow all languages',
+            action='store_true', default=False, dest='all_lang')
 
     def handle(self, *args, **options):
         problem = Problem()
         problem.code = options['code']
         problem.name = options['name']
         problem.description = options['body']
         problem.group = ProblemGroup.objects.get(name=options['group'])
-        problem.types = [ProblemType.objects.get(name=options['type'])]
+        problem.time_limit = options["time_limit"]
+        problem.memory_limit = options["memory_limit"]
+        problem.points = options["points"]
+        if options['pr_pts']:
+            problem.partial = 1
+        problem.save()
+
+        problem.types.set([ProblemType.objects.get(name=options['type'])])
+        if options['all_lang']:
+            problem.allowed_languages.set([l for l in Language.objects.all()])
```

Comment:
```suggestion
            problem.allowed_languages.set(Language.objects.all())
```
This should work and removes the unnecessary list comprehension.

Reply:
Tested it works fine. Thanks.